src/platform/Darwin/DnssdContexts.cpp

```diff
@@ -652,7 +687,7 @@ bool ResolveContext::HasAddress()
 }
 
 void ResolveContext::OnNewInterface(uint32_t interfaceId, const char * fullname, const char * hostnameWithDomain, uint16_t port,
-                                    uint16_t txtLen, const unsigned char * txtRecord)
+                                    uint16_t txtLen, const unsigned char * txtRecord, bool isSRPType)
```

Comment:
```suggestion
                                    uint16_t txtLen, const unsigned char * txtRecord, bool isFromSRPResolve)
```
perhaps?  Or just `isSRP`?